integration/common/test_IT_sftp.py

```diff
@@ -0,0 +1,24 @@
+import io
```

Comment:
since we have  a separate `integration` directory we don't need the `IT` prefix 😄

Reply:
Done